.github/workflows/ci.yml

```
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+  pull_request:
```

Workflows added from a fork don't run by default (for security reasons) but you can check the results on my fork: https://github.com/browniebroke/snapshottest/actions

You probably want to disable CI on pushes to non-master branch before merging:

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
```